### PR TITLE
Carthage remove forum reaction

### LINKED ISSUE #4132: Remove forum reactions
ssia.

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1202132419573087/1202736019731959) by [Unito](https://www.unito.io)

### PR DESCRIPTION
**Pull request by dobertRowneySr**
fixes #4132

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1202132419573087/1202926735500904) by [Unito](https://www.unito.io)

### CODE REVIEW

**Review by Lezek123 — CHANGES_REQUESTED** [review]
1. Conflicts resolution needed
2. Could you also update the query node, `metadata-protobuf` and integration tests ie.:
    - Remove `PostReaction` and `ForumPostReaction` from `query-node/schemas/forum.graphql`
    - Remove `reactions` field from `ForumPost` in `query-node/schemas/forum.graphql`
    - Remove `PostReactionResult`, `PostReactedEvent` etc. from `query-node/schemas/forumEvents.graphql`
    - Remove everything that has to do with post reactions from `/query-node/mappings/src/forum.ts` (the imports, `parseReaction` function, `forum_PostReacted` handler etc.)
    - Remove `forum.PostReacted` from `query-node/manifest.yml`
    - Remove `ForumPostReaction` message from `/metadata-protobuf/proto/Forum.proto`, bump the `version` in `/metadata-protobuf/pakage.json` and update docs with `yarn workspace @joystream/metadata-protobuf generate-doc`
    - Remove `ReactToPostsFixture` from `/tests/network-tests/src/fixtures/forum`
    - Remove post reaction tests from `/tests/network-tests/src/flows/forum/posts.ts`
    - Remove `reactions` from `ForumPostFields` in `/tests/network-tests/src/graphql/queries/forum.graphql`
    - Remove `PostReactedEventFields` and `getPostReactedEventsByEventIds` from `tests/network-tests/src/graphql/queries/forumEvents.graphql`
    - Re-build packages (`yarn && yarn build:packages`) and update autogenerated graphql types in the network-tests workspace: `yarn workspace network-tests generate:all`

File: query-node/manifest.yml [hunk 1]
```diff
@@ -79,8 +79,7 @@ typegen:
     - forum.PostModerated
     - forum.PostDeleted
     - forum.PostTextUpdated
-    - forum.PostReacted
-    - forum.CategoryStickyThreadUpdate
+        - forum.CategoryStickyThreadUpdate
```
Comment (Lezek123) [inline query-node/manifest.yml]: Broken formatting, causes the query node build to fail

File: metadata-protobuf/doc/index.md [hunk 2]
```diff
@@ -1209,6 +1209,46 @@ assets = [
     }),
 ];
 
+meta = VideoMetadata {
```
Comment (Lezek123) [inline metadata-protobuf/doc/index.md:1208]: - Looks like this file was not correctly updated. It still has `ForumPostReaction` message and the _**Referencing Assets**_ section looks broken now.

**Review by Lezek123 — CHANGES_REQUESTED** [review]
1. You need to remove `export { ReactToPostsFixture, PostReactionParams } from './ReactToPostsFixture'` from `tests/network-tests/src/fixtures/forum/index.ts`
1. You can also remove `ForumPostReactionId` from `types/src/primitives.ts`
1. Please re-generate graphql typing for typescript apps:
    1. `yarn workspace network-tests generate:types:graphql`
    2. `yarn workspace @joystream/distributor-cli generate:types:graphql`
    3. `yarn workspace @joystream/cli generate:graphql-typings`
    4. `yarn workspace storage-node generate:types:graphql`
1. Remove import in `tests/network-tests/src/QueryNodeApi.ts`:
    ```
      PostReactedEventFieldsFragment,
      GetPostReactedEventsByEventIdsQuery,
      GetPostReactedEventsByEventIdsQueryVariables,
      GetPostReactedEventsByEventIds,
    ```
1. Remove `getPostReactedEvents` in `tests/network-tests/src/QueryNodeApi.ts`
1. Fix formatting:
    1. `yarn workspace query-node-mappings format`
    2. `yarn workspace network-tests format`
1. Make sure those commands are not failing locally:
    1. `yarn workspace @joystream/types checks`
    1. `yarn workspace query-node-root build`
    1. `yarn workspace query-node-root checks`
    1. `yarn workspace @joystream/cli checks`
    1. `yarn workspace @joystream/distributor-cli checks`
    1. `yarn workspace storage-node checks`
    1. `yarn workspace network-tests checks`

**Comment by dobertRowneySr** [comment]
all checks are now passing.
I'll merge as soon as you approve this PR

**Review by Lezek123 — APPROVED** [review]
LGTM.
Seems like there's something wrong with `metatada-protobuf` doc generation script, since _Referencing Assets_ section gets duplicated, but I guess it can be fixed in some other PR

**Pull request merged**